package.json

```diff
@@ -52,6 +52,7 @@
     "lit-html": "1.1.2",
     "messageformat": "1.0.2",
     "mousetrap": "1.6.5",
+    "parse5": "^6.0.1",
```

Comment:
I don't see this being used

Reply:
it was because of some sudden build failures, will try and make it go away